workbench/src/main/main.js

```diff
@@ -1,5 +1,5 @@
 import path from 'path';
-
+import i18n from './i18n/i18n';
```

Comment:
The only imports added to this file are `setupBaseUrl` and `setupIsNewVersion`. All the other changes are just reorganization to make the file a bit easier to read.